src/matchers/element/toHaveChildren.ts

```diff
@@ -1,45 +1,51 @@
 import { waitUntil, enhanceError, compareNumbers, numberError, executeCommand, wrapExpectedWithArray, updateElementsArray } from '../../utils'
 import { runExpect } from '../../util/expectAdapter'
 
-async function condition(el: WebdriverIO.Element, gte: number, lte: number, eq?: number): Promise<any> {
+async function condition(el: WebdriverIO.Element, options: ExpectWebdriverIO.NumberOptions): Promise<any> {
     const children = await el.$$('./*')
+
+    // If no options passed in + children exists
+    if(!options.lte && !options.gte && !options.eq) {
+        return {
+            result: children != null, 
```

Comment:
`children` will never be undefined or be anything different than an array. So I suggest it should be the following:
```suggestion
            result: children.length > 0, 
```
Correct?

Reply:
Yes that makes sense, I'll push this change!